### PR TITLE
Add new metrics

### PR DESCRIPTION
**Pull request by ninjascant**
## Changes

* Update `DeFi` metric section: add `inflow_per_label_and_owner`, `outflow_per_label_and_owner`, `balance_per_label_and_owner_delta` and `balance_per_owner` metrics
* Add `MakerDAO` metric section
* Add `DEX` metric section
* Add `ETH 2.0` metric section
* Add `Labeled Balance` section
* Add `Labeled Exchanges` section
* Add `Labeled Flow` section

## Checklist

- [x] My article available in Navigation sidebar and in root article of selected category ([how to do it in README](https://github.com/santiment/academy#add-an-article-into-navigation-sidebar))
- [x]	My article have [metadata](https://github.com/santiment/academy#metadata) (title, description, date when updated and author)
- [x]	All added/updated links in article are exist, images shows correctly

### CODE REVIEW

File: src/docs/metrics/defi/index.md [hunk 1]
```diff
@@ -42,13 +50,11 @@ Dollars/Amount of coins
 ---
 
 ## Available Assets
-
-Available for `ethereum`.
+Available for `ethereum` and `usd`.
```
Comment (IvanIvanoff) [inline src/docs/metrics/defi/index.md:53]: What is the `usd` asset? Should this be an asset/currency pair?

File: src/docs/metrics/dex/index.md [hunk 2]
```diff
@@ -0,0 +1,323 @@
...
+
+Segmented by DEX and asset:
+* `eth_trade_volume_by_token` - Volume of trades between ETH-Based tokens and a given ERC20 token
+* `stablecoin_trade_volume_by_token` - Volume of trades between Stablecoins\*\* and a given ERC20 token
+* `token_eth_price_by_dex_5m` - Token ETH price segmented by DEX for trades between ETH-Based tokens\* and a given 
+  ERC20 token
+
+---
+_\* ETH-based tokens - ETH, ETH Kyber, WETH, ETHWrapped and Bancor ETH._   
+_\*\* USD-based Stablecoins - Tether, USD coin, DAI, Binance USD, TrueUSD and etc._
```
Comment (IvanIvanoff) [inline src/docs/metrics/dex/index.md:31]: You can try using:
```md
<sup>[1](#eth-based-tokens)</sup>
```
 and follow it with
```md
### ETH Based Tokens
ETH, ETH Kyber, WETH, ETHWrapped and Bancor ETH.
```

Comment (Snedashkovsky) [reply src/docs/metrics/dex/index.md]: Added

File: src/docs/metrics/defi/index.md (hunk 1 above)
Comment (Snedashkovsky) [reply src/docs/metrics/defi/index.md]: Fixed

**Comment by haritonasty** [comment]
@Snedashkovsky I've changed `eth-2` folder name because in navigation sidebar parsing file path had error

**Pull request merged**